### PR TITLE
README improvements

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,19 @@ When using a different port, please make sure that it is not used by [another se
...
 
+##### Quickstart for local development
+
+Run:
+1. `./gradlew buildAllImages`
+2. `ORT_SERVER_IMAGE_PREFIX= ORT_SERVER_IMAGE_TAG=latest ORT_SERVER_CORE_PORT=8090 docker compose up -d`
+3. `pnpm -C ui install`
+4. `pnpm -C ui build`
+5. `pnpm -C ui dev`
+6. Open http://localhost:5173/.
```
Comment (mmurto) [inline README.md:57]: Docker compose serves the UI on 8082, so the pnpm instructions could probably be deleted here?

Comment (sschuberth) [reply README.md:57]: Yes, true, except that I cannot get it to work, at least not when using a non-default `ORT_SERVER_CORE_PORT=8090`.

Comment (mmurto) [reply README.md:57]: Did you try setting [`UI_API_URL`](https://github.com/eclipse-apoapsis/ort-server/blob/main/ui/docker/entrypoint.sh#L34-L36) to `http://localhost:8090` for the UI container?

Comment (sschuberth) [reply README.md:57]: Yes. Maybe https://github.com/eclipse-apoapsis/ort-server/pull/528 is the missing bit.

Comment (mmurto) [reply README.md]: It sets it for the container, so looks good!

**Review by mmurto — APPROVED** [review]

**Pull request merged**